Formula/opal.rb

```diff
@@ -0,0 +1,31 @@
+class Opal < Formula
+  desc "Ruby to JavaScript transpiler"
+  homepage "https://opalrb.com/"
+  url "https://github.com/opal/opal.git",
```

Comment:
Do we need the Git clone?

Reply:
Yes, there's submodules. I initially tried with tarball and it just ended up building nothing.